store/server/features/leads/mutation.ts

```
@@ -0,0 +1,99 @@
+            }
+            return lead;
+          });
+          return updatedData;
+        }
+
+        return oldData;
+      });
+
+      return { previousLeads };
+    },
+    onSuccess: (data, variables) => {
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)